src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -27,36 +31,40 @@ class ReviewPage extends React.Component {
     focusElement('h2');
   }
 
+  renderDowntime = (downtime, children) => {
+    if (downtime.status === externalServiceStatus.down) {
+      const Message = this.props.formConfig.downtime.message || DowntimeMessage;
+
+      return <Message downtime={downtime} />;
+    }
+
+    return children;
+  };
+
   render() {
-    const { formConfig, pageList, path } = this.props;
+    const { formConfig, pageList, path } = this.props.route;
 
+    const downtimeDependencies = formConfig?.downtime?.dependencies || [];
```

Comment:
Comes from `RoutedSavableReviewPage`, only no `lodash`:

https://github.com/department-of-veterans-affairs/vets-website/blob/d412bc0a1834ff507761f4be7f4d94eebb876349/src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx#L81

Reply:
For what it's worth, `get` isn't _actually_ `lodash`.

https://github.com/department-of-veterans-affairs/vets-website/blob/dtn-reviewPage/src/platform/forms/save-in-progress/RoutedSavableReviewPage.jsx#L15
```js
import get from '../../utilities/data/get';
```

Using optional chaining is the right approach here, thought. :+1: